src/resources/scss/global.scss

```diff
@@ -55,4 +55,116 @@
             margin-left: 0;
         }
     }
+}
+
+.footer{
+    position: relative;
+    padding: ($includingSwooshPadding + 50px) 20px 0 20px;
+    background: $bio-gray;
+    color: $black;
+
+    @media screen and (min-width: $medium) {
+        display: flex;
+        justify-content: space-between;
+        padding: $includingSwooshPadding 10vw 0 10vw;
+    }
+    
+    @media screen and (min-width: $large) {
+        padding: $includingSwooshPadding 15vw 0 15vw;
+    }
+
+    @media screen and (min-width: $xlarge) {
+        padding: $includingSwooshPadding 18vw 0 15vw;
+    }
+
+    
+    
+    &__githubLogo{
+        position: absolute;
+        top: 30px;
+        left: 0;
+        transform: rotate(90deg);
+        width: 70px;
+
+        @media screen and (min-width: $large) {
+            top: auto;
+            left: auto;
+            bottom: 0;
+            transform: rotate(0);
+            width: auto;
+        }
+    }
+
+    &__checkItOut{
+        padding-top: 25px;
+        line-height: 1.3;
+
+        @media screen and (min-width: $medium) {
+            flex-grow: 1;
+            padding: 0 3vw;
+            font-size: 20px;
+        }
+        @media screen and (min-width: $large) {
+            padding-left: 17vw;
+        }
+    }
+    
+    &__bigWords{
+        font-family: $font-family-header;
+        font-size: 30px;
```

Comment:
Consider exporting this typeface to `typography.scss` please.

Reply:
Would this mean we move all the fonts from _settings.scss to typography.scss?

Reply:
No, that's just the place for extendable typefaces such as %benefit-text.